### DISCUSSION TITLE
Not working for me :(

### DISCUSSION THREAD

**Discussion by gitBellucci** [Zero6992/chatGPT-discord-bot#94] [Q&A] [answered]
"ChatGPT
BOT
 — Aujourd’hui à 06:05
Error: Something went wrong, please try again later!"

when using  /chat i'm getting this error : openai.error.RateLimitError: You exceeded your current quota, please check your plan and billing details.

Can anyone help please

**Comment by Jefw1f** [comment, marked as answer]
I solved this problem updating my account in open ai to a premium one.
